molgenis-core-ui/src/main/resources/js/jquery.molgenis.xrefmrefsearch.js

```diff
@@ -147,7 +147,7 @@
 			},
 			initSelection: function(element, callback) {
 				//Only called when the input has a value
-				var attrNames = (uniqueAttrNames.length ? uniqueAttrNames : lookupAttrNames);
+				var attrNames = uniqueAttrNames;
```

Comment:
I don't understand, why does it use uniqueAttrNames to begin with, instead of the ID column?